text/0517-io-os-reform.md

```diff
@@ -596,13 +616,17 @@ method, clients of `Read` can easily recover the `push` method.
 ### `Write`
 [Write]: #write
 
-The `Writer` trait is cut down to even smaller size:
+The `Writer` trait is cut down to even smaller size, and `flush` is taken
```

Comment:
`Writer` doesn't really get any smaller - all of the same methods are still there - just refactored.